### DISCUSSION TITLE
Errors continue to occur during GitHub Student Developer Pack authentication.

### DISCUSSION THREAD

**Discussion by galaxy-note7-explosion** [community/community#62679] [GitHub Education]
### Hi everyone,

제가 한국인이라서 영어를 잘 하질 못합니다. 
제가 잘못된 곳에 이 질문을 올렸었더라도 양해 부탁드립니다.

I am not good at English because I am Korean.
Please understand if I posted this question in the wrong place.

**error messege** : You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?
The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation.

The same error is repeated over and over again.
My student ID satisfies all the conditions, and I applied my name and short bio. I've met all those conditions, but I keep getting the same error.
I took pictures with my laptop, so the resolution is pretty good, but I don't know what's going on.
Especially, "If your original document is not in English then you might take a photo of the original next to an English translation." 
this part is related to English translation, but I don't understand how to translate it.

p.s.) i'm korean 

kor) 같은 오류가 계속해서 반복됩니다.
제 학생증은 조건을 모두 만족하고 있으며, 저의 이름과 짧은 바이오도 적용하였습니다. 저 조건을 모두 만족했는데도 계속해서 같은 오류가 뜹니다.
노트북 컴퓨터로 사진을 찍어서 해상도가 괜찮은 편인데 대체 왜 이러는지 알 수가 없네요

특히 " If your original document is not in English then you might take a photo of the original next to an English translation."
=> 어케 번역을 하라는지 이해가 전혀 안되네요.

**Comment by Danylo-Honcharenko** [comment]
> but I don't understand how to translate it.

Good day, please use Google Translate. Select **"Images"** and upload your photo by selecting the language you want to translate from and to which you want to translate _(English)_.

Next, upload the photo to your computer by clicking on the **"Download translation"** button. For convenience, you can also create a .txt file on your desktop, then click on the **"Copy Text"** button and paste the text there to copy the name into the **"Name"** field in GitHub. https://github.com/settings/profile

> i'm korean

좋은 하루 되세요. Google 번역을 사용하세요. 이미지를 선택하고 번역할 언어와 번역할 언어를 선택하여 사진을 업로드합니다(영어).

그런 다음 번역 다운로드 버튼을 클릭하여 컴퓨터에 사진을 업로드합니다. 편의를 위해 바탕 화면에 .txt 파일을 만든 다음 텍스트 복사 버튼을 클릭하고 거기에 텍스트를 붙여넣어 이름을 GitHub의 이름 필드에 복사할 수도 있습니다. https://github.com/settings/profile
